docs/self-managed/identity/user-guide/roles/add-assign-permission.md

```diff
@@ -16,16 +16,16 @@ You can create permissions for granular access control over your APIs. Permissio
-| Optimize    | `write:*`                                                   | Write access to all APIs                                                                                           |
-| Tasklist    | `read:*` <br/> `write:*`                                    | Read access to all APIs <br/> Write access to all APIs                                                             |
-| Web Modeler | `create:*` <br/> `read:*` <br/> `update:*` <br/> `delete:*` | CRUD access                                                                                                        |
-| Zeebe       | `write:*`                                                   | Write access to all APIs                                                                                           |
+| Component   | Permissions                                                                                 | Descriptions                                                                                                       |
+| ----------- | ------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------ |
+| Connectors  | `read:*`                                                                                    | Read access to all APIs                                                                                            |
+| Console     | `write:*`                                                                                   | Write access to all pages                                                                                          |
+| Identity    | `read` <br/> `read:users` <br/> `write`                                                     | Read access to all pages <br/> Access only the **Users** page and related subpages <br/> Write access to all pages |
+| Operate     | `read:*` <br/> `write:*`                                                                    | Read access to all APIs <br/> Write access to all APIs                                                             |
+| Optimize    | `write:*`                                                                                   | Write access to all APIs                                                                                           |
+| Tasklist    | `read:*` <br/> `write:*`                                                                    | Read access to all APIs <br/> Write access to all APIs                                                             |
+| Web Modeler | `write:*` <br/> `admin:*` <br/> `create:*` <br/> `read:*` <br/> `update:*` <br/> `delete:*` | Access to the Internal API <br/> Elevated Access <br/> CRUD access to Public API                                   |
```

Comment:
❓ Would it make sense to call this "Elevated Access to the Internal API"? For additional clarity, we could also link back to the super-user (and coming up the "publishing connector templates") page here.